packages/legend-studio/src/components/editor/edit-panel/element-generation-editor/FileGenerationEditor.tsx

```diff
@@ -217,6 +218,53 @@ export const GenerationResultExplorer = observer(
   },
 );
 
+export const visualizeMorphirButtonGetter = (
```

Comment:
`Morphir`/`Bosque` should not be mentioned in the legend-studio core code. They are plugins not part of the the core legend studio core. What exactly are these buttons represent for the file generation editor. They should be able to be implemented by any other type of file generation in the future

Reply:
Thanks! Agree -- refactored interface names to generic ones.